cmd/wallet-sdk-gomobile/api/api.go

```diff
@@ -56,15 +56,14 @@ type KeyHandleReader interface {
 
 // CreateDIDOpts represents the various options for the DIDCreator.Create method.
 type CreateDIDOpts struct {
-	// The type of DID to create (i.e. which DID method to use).
-	Method string
+	Key []byte
```

Comment:
I think key name is bit confusing may be keyType ??

Reply:
The `Key` there is the raw key bytes. However, I've updated my PR so that's no longer there - take a look and let me know what you think!